apps/melding-form/src/app/(map)/locatie/kies/_components/_utils/updateWfsLayer.test.ts

```diff
@@ -0,0 +1,74 @@
+import { buildWfsLayer } from './buildWfsLayer'
+import { updateWfsLayer } from './updateWfsLayer'
+
+vi.mock('./buildWfsLayer', () => ({
+  buildWfsLayer: vi.fn(),
+}))
+
+describe('updateWfsLayer', () => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let mapInstance: any
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let assetLayerRef: { current: any }
+
+  beforeEach(() => {
+    assetLayerRef = { current: null }
+    mapInstance = {
+      getZoom: vi.fn(),
+    }
+    vi.clearAllMocks()
+  })
+
+  it('returns undefined if mapInstance is falsy', () => {
+    updateWfsLayer(null as any, assetLayerRef, 'container')
```

Comment:
Our types already protect us against this, no? `mapInstance` cannot be null. Do we need the extra check?